### ISSUE TITLE
[sass|scss] max() function causing fatal error

### ISSUE THREAD

**Issue by EvilClosetMonkey** [sasstools/sass-lint#1087]
**What version of Sass Lint are you using?**
1.10.2

**What did you do? Please include the actual source code causing the issue.**

```scss
    @if (max(map_values($grid-breakpoints)...) == $_breakpoint) {
      @content;
    }
```

Where `$grid-breakpoints` is a map and `$_breakpoint` is pixel value (e.g., 1920px).

**What did you expect to happen?**

The SASS compiler accepts and compiles this; sass-lint should not flag it as invalid.

**What actually happened? Please include any error messages given to you by Sass Lint.**

  13:1  error  Please check validity of the block starting from line #13  Fatal

**Comment by DanPurdy** [comment]
The problem unfortunately lies with the current version of the AST we use. This is being updated as we speak and we will check back on specific extra errors once it's released.

**Comment by bgriffith** [comment]
This is fixed as of `v1.11.1`.
